app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -1,70 +1,63 @@
-            >
-              <td>
-                <Checkbox
-                  checked={selectedNoteIds.includes(note.id)}
-                  onClick={event => {
-                    event.stopPropagation();
-                    const index = selectedNoteIds.indexOf(note.id);
+import DeleteAlert from "./DeleteAlert";
 
-                    if (index > -1) {
-                      setSelectedNoteIds([
-                        ...selectedNoteIds.slice(0, index),
-                        ...selectedNoteIds.slice(index + 1)
-                      ]);
-                    } else {
-                      setSelectedNoteIds([...selectedNoteIds, note.id]);
-                    }
-                  }}
-                />
-              </td>
-              <td>
-                <div className="flex flex-row items-center justify-start text-gray-900">
-                  {note.title}
-                </div>
-              </td>
-              <td>{note.description}</td>
-            </tr>
-          ))}
-        </tbody>
-      </table>
+function NoteTable() {
+  return (
+    <div>
+      <Card />
+      <Card />
+      <Card />
+      <Card />
     </div>
   );
 }
+
+const Card = () => {
+  const [delToast, setDelToast] = React.useState(false);
+  const handleDelete = () => {
+    setDelToast(true);
+  };
+  return (
+    <div className="mb-4 rounded-sm neeto-ui-border-gray-300 neeto-ui-shadow-xs border neeto-ui-bg-white">
+      <div className="p-4">
+        <div className="flex justify-between">
+          <Typography style="h4">How to claim the warranty?</Typography>
+          <div className="flex flex-row items-center justify-end space-x-3">
+            <Dropdown icon={MenuVertical} buttonStyle="icon" autoWidth>
+              <li>Edit</li>
+              <li onClick={handleDelete}>Delete</li>
+            </Dropdown>
+          </div>
+        </div>
+        <Typography className="text-sm leading-5 neeto-ui-text-gray-600 font-normal border-b pb-4 neeto-ui-border-gray-300">
+          "Are you getting my texts???" she texted to him. He glanced at it and
+          chuckled under his breath. Of course he was getting them, but if he
+          wasn't getting
+        </Typography>
+        <div className="flex justify-between mt-4">
+          <Label className="border rounded-sm py-1 px-2 neeto-ui-bg-gray-100">
+            Getting Started
+          </Label>
+          <Label className="flex justify-evenly">
+            {<Clock width="16" />}{" "}
+            <Tooltip placement={"bottom-end"} content={"Wednesday, 10:30AM"}>
+              <p className="inline-block mx-1"> Created 2 hours ago</p>
+            </Tooltip>
+            <img
+              src="https://randomuser.me/api/portraits/women/90.jpg"
+              width="24"
+              className="inline-block rounded-full"
+            />
+          </Label>
+        </div>
+      </div>
+      <DeleteAlert showToast={delToast} setShowToast={setDelToast} />
+    </div>
```

Comment:
Since this is a reusable component, please split this as follows - 
1. Create a file called `index.jsx` inside a folder  called `Card`.
2. Each card should consist of `CardHeader, CardBody and CardFooter`.
3. Each component should be split into a different file and imported to `index.jsx`

Reply:
Also, since this is a reusable component, please define `prop-types` for it.